helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -29,6 +29,22 @@ data:
           enabled: '{{ .Values.Installation.V1.Provider.AWS.TrustedAdvisor.Enabled }}'
         vaultAddress: '{{ .Values.Installation.V1.Auth.Vault.Address }}'
         vpcPeerID: '{{ .Values.Installation.V1.Provider.AWS.VPCPeerID }}'
+      cluster:
+        calico:
+          cidr: '{{ .Values.Installation.V1.Guest.Calico.CIDR }}'
+          mtu: 1430
+          subnet: '{{ .Values.Installation.V1.Guest.Calico.Subnet }}'
```

Comment:
Due to the new type and the required type mapping we need to configure a couple of new information to the operator. This is all taken from `kubernetesd`. Check if this is fine. I need to adapt the e2e tests configuration for that because the operator fails now when started without sufficient configuration.

Reply:
On high level this seems correct. Didn't cross check for syntax specifically.